exts/betterTooltip/betterTooltip.js

```diff
@@ -73,6 +183,17 @@
                 return _categoryLabelFormatFunction;
             };
 
+            /**
+             * Gets or sets the format string for the label of the series. Used by the default label formater.
```

Comment:
Gets or sets the format string for the label of the series. Used by the default label formatter.